styles/_footer.scss

```diff
@@ -1,20 +1,195 @@
 /***** Footer *****/
 .footer {
-  border-top: 1px solid $border-color;
-  margin-top: 60px;
-  padding: 30px 0;
+  margin: 0 auto;
+  @include fluid-value( padding-top, $min-width, $max-width, $whitespace-medium, $whitespace-medium--desktop );
+  max-width: $max-width-page-container;
+  background-color: $primary--light;
+  color: $white;
 
-  a { color: $secondary-text-color; }
+  &__inner {
+    display: block;
+  }
```

Comment:
do you need this one? Adding a display block to a div

Reply:
Removed!